# HG changeset patch
# User Olli Kallioinen <olli.kallioinen@iki.fi>
# Date 1505306827 -10800
#      Wed Sep 13 15:47:07 2017 +0300
# Node ID 4e233d7c9a5c1376ae7f326c4d83d99559d2f20e
# Parent  209ac86c4e86f30ff3dcc9f9de12f02d0ba0b174
Android cmake build fix: Cannot use a postfix on debug version library names on android because the libraries are loaded explicitly by name.

diff -r 209ac86c4e86 -r 4e233d7c9a5c CMakeLists.txt
--- a/CMakeLists.txt	Tue Sep 12 05:53:47 2017 -0700
+++ b/CMakeLists.txt	Wed Sep 13 15:47:07 2017 +0300
@@ -48,8 +48,10 @@
 set(SDL_VERSION "${SDL_MAJOR_VERSION}.${SDL_MINOR_VERSION}.${SDL_MICRO_VERSION}")
 
 # Set defaults preventing destination file conflicts
-set(CMAKE_DEBUG_POSTFIX "d"
-    CACHE STRING "Name suffix for debug builds")
+if(NOT ANDROID)
+  set(CMAKE_DEBUG_POSTFIX "d"
+      CACHE STRING "Name suffix for debug builds")
+endif()
 
 mark_as_advanced(CMAKE_IMPORT_LIBRARY_SUFFIX CMAKE_DEBUG_POSTFIX)
 
